src/lib/y2storage/planned/lvm_vg.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env ruby
+#
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage/planned/device"
+require "y2storage/planned/mixins"
+
+module Y2Storage
+  module Planned
+    # Specification for Y2Storage::LvmVg object to be created during the
+    # storage or AutoYaST proposals
+    #
+    # @see Device
+    class LvmVg < Device
+      include Planned::HasSize
+
+      DEFAULT_VG_NAME = "system".freeze
+
+      DEFAULT_EXTENT_SIZE = DiskSize.MiB(4)
+      private_constant :DEFAULT_EXTENT_SIZE
+
+      # @return [String] name to use for Y2Storage::LvmVg#vg_name
+      attr_accessor :volume_group_name
+
+      # @return [Array<Planned::LvmLv>] List of logical volumes
+      attr_accessor :lvs
+
+      # @return [Array<Planned::Device>] List of physical volumes
+      attr_accessor :pvs
+
+      # @return [DiskSize] Size of one extent
+      attr_accessor :extent_size
+
+      # @return [DiskSize] Total size of the volume group
```

Comment:
It's strange to have a writer for this. I would expect that value to be automagically set for reused VGs (that is, as a consequence of `.from_real_vg`), but not to have a writer to set it by hand.